### DISCUSSION TITLE
How can I use react hooks feature with hexo-renderer-react plugin?

### DISCUSSION THREAD

**Discussion by nojsja** [hexojs/hexo#5141] [Question / Answer] [answered]
[hexo-renderer-react](https://www.npmjs.com/package/hexo-renderer-react)
I tried it and found that the compiled code just includes jsx dom content but some inner feature in React such as hooks are unavailable.

**Comment by stevenjoezhang** [comment, marked as answer]
hexo-renderer-react is a third-party plugin and is not updated for years. It might not support the new features.
